CustomDictionary.xml

```diff
@@ -30,20 +31,23 @@
       <Word>rev</Word>
       <Word>rfc</Word>
       <Word>src</Word>
-	  <Word>Cron</Word>
```

Comment:
I added some new terms and the rest was bad alignment that has been fixed -- I have VS set to auto-format on save so it aligns everything properly. Put ?w=1 at the end the url to have the diff ignore whitespace. It's a lot easier to review real changes that way.

---
In reply to: [54778349](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/45#discussion_r54778349) [](ancestors = 54778349)